content/docs/user-guide/troubleshooting.md

```diff
@@ -46,3 +46,8 @@ Make sure you are online and able to access your
 Make sure your bucket
 [exists](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-bucket.html)
 in the correct `region` and/or `endpointurl` (see `dvc remote modify`).
+
+## Unable to detect cache type {#unable-to-detect-cache-type}
+
+Unable to detect supported link types, as cache directory `.dvc/cache` doesn't exist. It is usually auto-created by commands such as `dvc add/fetch/pull/run/import`, but you could create it manually to enable this check.
```

Comment:
Thanks! A bit confused. Is the problem that the cache doesn't exist yet, or that the cache type can't be detected? I get that the former causes the latter, but they're also separate situations no? For example, maybe a corrupted DVC cache or bad config file could also cause an inability to detect the type.

Reply:
> For example, maybe a corrupted DVC cache or bad config file could also cause an inability to detect the type.

@jorgeorpinel Not really, we don't use cache files or configs. The only issue here is nonexistent cache dir, that's it.

Reply:
OK. Still seems kind of indirect but if that's the only possible scenario for this issue, then fine 🙂